libcontainer/rootfs_linux.go

```diff
@@ -867,7 +867,7 @@ func msMoveRoot(rootfs string) error {
 }
 
 func chroot(rootfs string) error {
-	if err := unix.Chroot("."); err != nil {
+	if err := unix.Chroot(rootfs); err != nil {
```

Comment:
needs validation?

Reply:
should error out if rootfs != "." ?

Reply:
@AkihiroSuda Do you mean something like this?(check last commit)

Reply:
`if filepath.Clean(rootfs) != os.Getwd(){ return errors.New(..) }`? (Make sure to test it, ideally in CI)

Reply:
os.Getwd() returns absolute wd path, on other hand rootfs path here can be relative and we will get error even if they are point at same dir.

Reply:
`if filepath.Abs(rootfs) != os.Getwd(){ return errors.New(..) }`. The argument of `Chroot()` should remain `"."`.